docker_compose/.env

```diff
@@ -5,10 +5,16 @@ SCHEDULER_CONFIG_FILE_ABSOLUTE_PATH=
 TRAPS_CONFIG_FILE_ABSOLUTE_PATH=
 INVENTORY_FILE_ABSOLUTE_PATH=
 COREFILE_ABS_PATH=
+SC4SNMP_VERSION="1.12.1"
```

Comment:
latest

Reply:
it's changed automatically when we are releasing to the newest